backend/configuration/example.yaml

```
@@ -0,0 +1,17 @@
+# To use custom server settings for a local or a production environment create a copy
+# of this example file and name it local.yaml
+server:
+  host: 127.0.0.1
+  app_url: 'http://localhost:3007'
+dgraph:
+  host: 'localhost'
```

```suggestion
  host: 'http://localhost'
```

My tests were failing from this...

---

3f9cd65